### DISCUSSION TITLE
Discogs genre tags feature

### DISCUSSION THREAD

**Discussion by nmycps** [Marekkon5/onetagger#335] [Ideas]
Looking at how to get more accurate genre descriptions I have tested grabbing them from all the available sources on One Tagger. Still in my opinion Discogs has better descriptions. But what is called sub-genre as an option is called 'Style' in Discogs. But since my audio files mostly consist of electronic music, grabbing the 'Electronic' genre isn't my concern, but 'Style' is (which is created as a new tag). My question is, is it possible to tell One Tagger to place the 'Style' tag into the 'Genre' tag when using the Discogs option? I don't want to append it to 'Electronic' but rather replace it.

**Comment by bascurtiz** [comment]
> My question is, is it possible to tell One Tagger to place the 'Style' tag into the 'Genre' tag when using the Discogs option? 

Yes!
In One Tagger's AUTO TAG - at last section - Advanced,  scroll to bottom and change `Default` to `Write Style to Genre tag` from the dropdown-menu.

![image](https://github.com/Marekkon5/onetagger/assets/34075088/e0276f5d-1bad-46db-bb94-5c96993021f1)

**Comment by nmycps** [comment]
I had overlooked that feature! Thanks!
